### DISCUSSION TITLE
After update i need to reconfigure all containers

### DISCUSSION THREAD

**Discussion by MajorLOL** [qdm12/gluetun#1544] [Q&A]
Greetings,

After an update i need to reconfigure all my containers again that were connected to Gluetun,
Is there a solution for this so they will auto restart with Gluetun enabled automaticly?

With kind regards,
Dennis

**Comment by MajorLOL** [comment]
Also see post:
https://github.com/qdm12/deunhealth/issues/11

No solution on there tho, Anyone have other info i can use?

**Reply by ghost** [reply]
I use
network_mode: "container:gluetun"
in my Docker-compose, so it works for me
